ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureV2ButtonCustomization.java

```diff
@@ -12,27 +12,66 @@
  */
 public class ThreeDSecureV2ButtonCustomization extends ThreeDSecureV2BaseCustomization implements Parcelable {
 
-    private String backgroundColor;
```

Comment:
We opted to remove this internal member variables across all the customization classes in favor of getting/setting the actual customization features on the cardinal customization classes.